tests/pgsql/pgsql-7000-ips/README.md

```diff
@@ -0,0 +1,21 @@
+# Description
+
+Showcase engine behavior with simple pgsql stream rules and alert metadata, in
+IPS mode.
+
+## Behavior
+
+The 2 rules will match on the PGSQL queries for `SELECT` (sid 1) and `DELETE FROM`,
+(sid 2). However, the combination of IPS mode matching on the traffic as soon as
+the engine sees it with the continuously available stream buffer leads to way
+more matches than could be initially expected. The `payload_printable` EVE field
```

Comment:
if there are way more matches, why does the test fail with `expected 1 got 0`.. messages?

Reply:
Sorry for being unclear.
What I mean is that if there are more matches then why does the sub test to check for 37 alerts not fail with `expected 37 got 100..` or something?

Reply:
Ah! I decided to do a test that showcases what we currently have. 🤔

Reply:
hmm. I think the tests should always have the expected behavior so the PR corresponding to it shows that it is bringing in the expected behavior.

Even more confused now. If this is showing current behavior, why does anything fail at all?

Reply:
> hmm. I think the tests should always have the expected behavior so the PR corresponding to it shows that it is bringing in the expected behavior.

Ok, I can update/change them. Then we would have tests failing until we have a bug fix - in which case I'd like to first confirm that there's a bug, so a ticket is created, then I would also probably split this PR to remove the tests that show the bug 🤔 

> Even more confused now. If this is showing current behavior, why does anything fail at all?

Current behavior with the Suricata PR '-'

Reply:
>  then I would also probably split this PR to remove the tests that show the bug 🤔

indeed. that sounds better.
 
> Current behavior with the Suricata PR '-'

ah sorry, misunderstood. 😓